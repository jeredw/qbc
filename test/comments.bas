' this is a comment
rem this is also a comment: so is this: and this: and this
print 42 ' yep also a comment
