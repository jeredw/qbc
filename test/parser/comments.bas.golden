(program \n 
  (statement 
    (rem_statement rem)) \n 
  (statement 
    (print_statement print 
      (expr 
        (literal 42)))) \n <EOF>)
